<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetV4PoolDialog</class>
 <widget class="QDialog" name="NetV4PoolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>IPv4 Network Pool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../../image.qrc">
    <normaloff>:/Image/network.png</normaloff>:/Image/network.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="pix">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../../image.qrc">:/Image/network.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchBtn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Search</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../image.qrc">
         <normaloff>:/Image/Search.png</normaloff>:/Image/Search.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Add New Network</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../image.qrc">
         <normaloff>:/Image/add.png</normaloff>:/Image/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="ipTV">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionEditNetwork">
   <property name="icon">
    <iconset resource="../../../../../image.qrc">
     <normaloff>:/Image/edit.png</normaloff>:/Image/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit Network</string>
   </property>
   <property name="toolTip">
    <string>Edit Network</string>
   </property>
  </action>
  <action name="actionRemveNetwork">
   <property name="icon">
    <iconset resource="../../../../../image.qrc">
     <normaloff>:/Image/remove.png</normaloff>:/Image/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remve Network</string>
   </property>
   <property name="toolTip">
    <string>Remve Network</string>
   </property>
  </action>
  <action name="actionSubNetworkPool">
   <property name="icon">
    <iconset resource="../../../../../image.qrc">
     <normaloff>:/Image/subnet.png</normaloff>:/Image/subnet.png</iconset>
   </property>
   <property name="text">
    <string>View Subnet Pool</string>
   </property>
   <property name="toolTip">
    <string>Subnet Pool</string>
   </property>
  </action>
  <action name="actionNetwork_Assignment">
   <property name="icon">
    <iconset resource="../../../../../image.qrc">
     <normaloff>:/Image/connection.png</normaloff>:/Image/connection.png</iconset>
   </property>
   <property name="text">
    <string>Assign Network</string>
   </property>
   <property name="toolTip">
    <string>network Assignment</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../../../../image.qrc">
     <normaloff>:/Image/refresh.png</normaloff>:/Image/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionExport_PDF">
   <property name="icon">
    <iconset resource="../../../../../image.qrc">
     <normaloff>:/Image/pdf.png</normaloff>:/Image/pdf.png</iconset>
   </property>
   <property name="text">
    <string>Export PDF</string>
   </property>
   <property name="toolTip">
    <string>Export PDF</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>addBtn</tabstop>
  <tabstop>searchBtn</tabstop>
  <tabstop>ipTV</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../image.qrc"/>
 </resources>
 <connections/>
</ui>
